tests/suites/target_test.function

```diff
@@ -342,7 +342,7 @@ int execute_tests( int args, const char ** argv )
     void ** params = NULL;
     uint8_t * data = NULL, * p = NULL;
 
-    GREENTEA_SETUP( 180, "mbedtls_test" );
+    GREENTEA_SETUP( 800, "mbedtls_test" );
```

Comment:
What's the smallest timeout that works today for the platforms in our CI? 800 seems really long. Over 10 minutes!

Reply:
yes, but for NRF52840, when MBEDTLS_SHA1_C and MBEDTLS_GENPRIME are enabled, some tests finsh after more than 700 seconds.
Perhaps since these are not default in Mbed OS, we may consider reducing the timeout (as commented in https://github.com/ARMmbed/mbedtls/issues/2194#issuecomment-441419377 )

Reply:
For clarification, when I use the default configuration in Mbed OS, on NRF52840_DK target, except changing `MBEDTLS_MPI_MAX_SIZE` to 512, these test suite skip their tests(They are either dependent on `MBEDTLS_SHA1_C`, `MBEDTLS_GENPRIME` or both), and the longest one of them is ~270 seconds.
The question is whether we want to be able to test the non default configuration